modules/ts/src/ts_gtest.cpp

```diff
@@ -4054,7 +4054,7 @@ enum GTestColor {
   COLOR_YELLOW
 };
 
-#if GTEST_OS_WINDOWS && !GTEST_OS_WINDOWS_MOBILE
+#if GTEST_OS_WINDOWS && !GTEST_OS_WINDOWS_MOBILE && !defined WINRT
```

Comment:
@alalek, thank you for creating of this update. However I think there is no need to apply this to the current PR now